strings/strings.c

```diff
@@ -20,7 +24,15 @@ int string_length(char s[])
 */
 char *reverse_string(char rv[], char s[])
 {
-
+  int len = string_length(s);
+  printf("%d << length \n", len);
+  int rv_cnt = 0;
+  for(int i=len-1; i >= 0; i--){
+    rv[rv_cnt] = s[i];
+    rv_cnt++;
+  }
+  printf("%s \n", rv);
+  return rv;
```

Comment:
The test doesn't appear to be passing for this. Perhaps there's a character missing in `rv`?

Reply:
Particularly, does the `rv` string have a terminating null character (`'\0'`) as the last character?